scene/main/viewport.cpp

```diff
@@ -3303,6 +3321,7 @@ void Viewport::_bind_methods() {
 
 	ClassDB::bind_method(D_METHOD("_gui_show_tooltip"), &Viewport::_gui_show_tooltip);
 	ClassDB::bind_method(D_METHOD("_gui_remove_focus"), &Viewport::_gui_remove_focus);
+	ClassDB::bind_method(D_METHOD("_gui_remove_focus_for_window"), &Viewport::_gui_remove_focus_for_window);
```

Comment:
It seems both `_gui_show_tooltip` and `_gui_remove_focus` don't need to be bound, nothing seems to depend on them being bound (unless they're meant to be used from scripting, but then they shouldn't start with `_`).

Reply:
`_gui_remove_focus` is called by name here:

https://github.com/godotengine/godot/blob/5f2295f2dfdbf1ac4e63b562e4cef71f2fda4036/scene/main/viewport.cpp#L2470

Reply:
Fixed.

Reply:
@bruvzg Not after this commit :)